apps/service/src/components/FloatingButtons.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from "@emotion/styled";
+import { Button, ButtonHierarchy } from "design-system";
+import React from "react";
+
+interface FloatingButtonsProps extends React.ComponentProps<"div"> {
+  handleClickLeftButton?: () => void;
+  handleClickRightButton?: () => void;
```

Comment:
```suggestion
  handleClickLeftButton?: (
    e: React.MouseEvent<HTMLButtonElement, MouseEvent>
  ) => void;
  handleClickRightButton?: (
    e: React.MouseEvent<HTMLButtonElement, MouseEvent>
  ) => void;
```

event 를 주는걸로 통일하면 좋을 것 같아요!